packages/unpublished-fastboot-test-app/config/ember-try.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+module.exports = function() {
+  return {
+    useYarn: true,
+    scenarios: [
+      {
+        name: 'fastboot-with-ember-fetch',
+        env: {
+          EMBER_OPTIONAL_FEATURES: JSON.stringify({ 'jquery-integration': false }),
```

Comment:
The ember-try config for -ember-data doesn't set `jquery-integration` to false. Is that difference on purpose?

Reply:
ill clean that up so it’s clear what we are testing there. Main thread has 4 variants to test where fastboot has 2.

- native fetch
- ember fetch jquery true
- ember fetch jquery false
- jquery true no ember fetch

Reply:
We only care about the outcome (does it work with jquery, does it work with fetch) in fastboot.

Reply:
but none of the main thread variations set jquery-integration to false? https://github.com/emberjs/data/blob/master/packages/-ember-data/config/ember-try.js

Reply:
@igorT we default jquery integration to false throughout the project, but we should make it explicit in the tests so that we don't accidentally get trolled.

Reply:
Yeah, my comment was that it seemed that here we hard code to false, but we don't in other similar places

Reply:
Made the scenarios explicit